src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.tweesky.cloudtools.codegen;
```

Comment:
@gcatanese I moved to something more generic, which should be better. 

However, it does modify the behaviour somehow (for example "success" default cases).

can you confirm whether that is desired?

Reply:
Looks good, an unexpected status code thrown an exception, which is probably better.

Reply:
ah, forgot that case, I'll an extra test!